client/components/data/query-user-purchases/index.js

```diff
@@ -0,0 +1,36 @@
+/** @format */
```

Comment:
Moved from `jsx` version.

Drops state listening on `hasLoadedUserPurchasesFromServer` and `isFetchingUserPurchases`. Rely on data-layer deduping for this.